frontend/src/components/shared/Dropdown/VirtualDropdown.tsx

```diff
@@ -48,7 +46,9 @@ function FixedSizeListItem(props: FixedSizeListItemProps): ReactElement {
       style={style}
       {...restChildProps}
     >
-      <StyledTruncateText>{item.label}</StyledTruncateText>
+      <OverflowTooltip content={item.label} placement={Placement.AUTO}>
```

Comment:
I'm a bit worried that `OverflowTooltip` is used slightly out of context here since it is not really used as a tooltip, or? It might work fine right for now, but in case we want to do changes to our actual tooltips, this might have negative consequences. Do we need the full `OverflowTooltip` here, or can we just extract the styled component and add it to the `Dropdown` component? I'm a bit undecided if it is fine to use it this way and in case it is easy to extract, we might want to do this.

Reply:
Thanks for taking a look! With `not really used as a tooltip` you mean that is not used like below?

![Screen Shot 2022-06-07 at 4 43 34 PM](https://user-images.githubusercontent.com/34423371/172468730-3d6c4590-b025-4a70-a2e7-15108b8a3210.png)

I've followed this direction because it's the recommended approach on the Github issue:

> Add a tooltip in case the text is getting truncated.

And because @tvst mentioned we were already doing that in S4T as well: https://github.com/streamlit/s4t/blob/8753a0cc1b9d56e5410afe8d2af314f8c73cdea0/web/src/components/tooltip/overflowTooltip.tsx

But happy to explore other options if we feel this might not be the best way to solve this. Let me know!

Reply:
Ahh, sorry. I think I misinterpreted the implementation. There is actually a tooltip added :) I thought it just adds the scrolling on mobile instead of ellipsis. I think this is good to be merged 👍